clear-all-data.sh

```diff
@@ -7,55 +7,70 @@
 #
 # Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
 # graphic logo are (registered/a) trademark(s) of Plan International.
-if [ "$REPLICAS" = "0" ]; then
-  HOST=mongo1
-  NETWORK=opencrvs_default
-  echo "Working with no replicas"
-elif [ "$REPLICAS" = "1" ]; then
-  HOST=rs0/mongo1
-  NETWORK=opencrvs_overlay_net
-  echo "Working with 1 replica"
-elif [ "$REPLICAS" = "3" ]; then
-  HOST=rs0/mongo1,mongo2,mongo3
-  NETWORK=opencrvs_overlay_net
-  echo "Working with 3 replicas"
-elif [ "$REPLICAS" = "5" ]; then
-  HOST=rs0/mongo1,mongo2,mongo3,mongo4,mongo5
-  NETWORK=opencrvs_overlay_net
-  echo "Working with 5 replicas"
-else
-  echo "Script must be passed an understandable number of replicas: 0,1,3 or 5"
-  exit 1
-fi
+print_usage_and_exit () {
+    echo 'Usage: ./clear-all-data.sh --path_to_core=XXX'
+    echo "This script clears all OpenCRVS data locally.  To run it you must pass the directory path to opencrvs-core."
+    echo "cd into opencrvs-core and run the command pwd to get this value."
+    echo "  --path_to_core must be subitted.  This is the path to your opencrvs-core directory with no trailing slash. E.G. /home/root/Documents/opencrvs-core"
+    exit 1
+}
+
```

Comment:
@Zangetsu101 these scripts are only run locally, so we do not need the REPLICAS, HOST and NETWORK blocks